components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/store/JDBCIdentityDataStore.java

```diff
@@ -266,6 +273,62 @@ public void remove(String userName, UserStoreManager userStoreManager) throws Id
         }
     }
 
+    @Override
+    public List<String> list(String claimUri, String claimValue,
+                             org.wso2.carbon.user.core.UserStoreManager userStoreManager) throws IdentityException {
+
+        List<String> userNames = new ArrayList<>();
+
+        // This is to support LDAP like queries, so this will provide support for only leading or trailing '*'
+        // filters. if the query has multiple '*' s the filter will ignore all.
+        if (claimValue.contains(QUERY_FILTER_STRING_ANY)) {
```

Comment:
Can't we support `*foo*` type of search filter ? This is already supported when searching by other claims.

Reply:
Yes this is supported in this logic.

Reply:
No its not supported in the current logic. Even the JDBC userstore manager does not support this. We have to fix both places if we are to support it.

Reply:
Created https://wso2.org/jira/browse/IDENTITY-6467 to track this seperately